2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,21 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    // for storing averages of each company's stock over 5 days
+    let averagePriceOverFiveDays = [ ];
+
+    // This loop cycles through each stock, the i is the 5 day stock for AAPL, MSFT etc.
+    for (i = 0; i < closingPricesForAllStocks.length; i++) {
```

Comment:
Issue: The iterator variable `i` in a for loop should be declared with `let`
Suggestion: Use a for...of loop instead of a for loop to improve readability